style/javascript/README.md

```diff
@@ -0,0 +1,24 @@
+JavaScript
+==========
+
+[Sample](sample.js)
+
+* Prefer ES6 classes over library provided classes.
+* Use strict equality checks (`===` and `!==`) except when comparing against
+  (`null` or `undefined`).
+* Prefer [arrow functions] `=>`, over the `function` keyword except when
+  defining classes or methods.
+* Use semicolons at the end of each statement.
+* Prefer double quotes.
+* Use `PascalCase` for classes, `lowerCamelCase` for variables and functions,
+  `SCREAMING_SNAKE_CASE` for constants, `_singleLeadingUnderscore` for private
+  variables and functions.
+* Prefer [template strings] over string concatenation.
+* Prefer promises over callbacks.
+* Prefer array functions like `map` and `forEach` over `for` loops.
+* Use `const` to declare variables.
```

Comment:
I think it's more "Use `const` when you don't have to rebind the value, otherwise use `let` and avoid `var`" so I'm having a hard time making it a single rule.